app/views/bgeigie_imports/_show.html.erb

```diff
@@ -71,4 +71,4 @@
   </div>
 </div>
 
-<%= render "delete" if current_user == @bgeigie_import.user || current_user.try(:moderator?) %>
+<%= render "delete" if !@bgeigie_import.approved? && (current_user == @bgeigie_import.user || current_user.try(:moderator?)) %>
```

Comment:
Every time I see these long authorization conditionals I wonder if we should set up cancan or something